src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/EmailController.php

```diff
@@ -0,0 +1,274 @@
+
+        $testEmailSendingForm = $this->createForm(TestEmailSendingType::class);
+
+        $configuration = $this->get('prestashop.adapter.legacy.configuration');
+        $isEmailLogsEnabled = $configuration->get('PS_LOG_EMAILS');
+
+        $presentedEmailLogsGrid = null;
+
+        if ($isEmailLogsEnabled) {
+            $gridPresenter = $this->get('prestashop.core.grid.presenter.grid_presenter');
+            $emailLogsGridFactory = $this->get('prestashop.core.grid.factory.email_logs');
+            $emailLogsGrid = $emailLogsGridFactory->createUsingSearchCriteria($filters);
+            $presentedEmailLogsGrid = $gridPresenter->present($emailLogsGrid);
+        }
+
+        return $this->render('@PrestaShop/Admin/Configure/AdvancedParameters/Email/email.html.twig', [
+            'emailConfigurationForm' => $emailConfigurationForm->createView(),
+            'isOpenSslExtensionLoaded' => $extensionChecker->loaded('openssl'),
+            'smtpMailMethod' => MailOption::METHOD_SMTP,
+            'testEmailSendingForm' => $testEmailSendingForm->createView(),
+            'emailLogsGrid' => $presentedEmailLogsGrid,
+            'isEmailLogsEnabled' => $isEmailLogsEnabled,
+            'enableSidebar' => true,
+            'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+        ]);
+    }
+
+    /**
+     * @AdminSecurity("is_granted('read', request.get('_legacy_controller')~'_')", message="Access denied.")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function searchAction(Request $request)
+    {
+        $definitionFactory = $this->get('prestashop.core.grid.definition.factory.email_logs');
+        $emailLogsDefinition = $definitionFactory->create();
+
+        $gridFilterFormFactory = $this->get('prestashop.core.grid.filter.form_factory');
+        $filtersForm = $gridFilterFormFactory->create($emailLogsDefinition);
+        $filtersForm->handleRequest($request);
+
+        $filters = [];
+
+        if ($filtersForm->isSubmitted()) {
+            $filters = $filtersForm->getData();
+        }
+
+        return $this->redirectToRoute('admin_email', ['filters' => $filters]);
+    }
+
+    /**
+     * Process email configuration saving
+     *
+     * @DemoRestricted(redirectRoute="admin_email")
+     * @AdminSecurity("is_granted(['update', 'create', 'delete'], request.get('_legacy_controller')~'_')", message="Access denied.")
+     *
+     * @param Request $request
+     *
+     * @return RedirectResponse
+     */
+    public function processFormAction(Request $request)
+    {
+        $formHandler = $this->getEmailConfigurationFormHandler();
+        $emailConfigurationForm = $formHandler->getForm();
+        $emailConfigurationForm->handleRequest($request);
+
+        if ($emailConfigurationForm->isSubmitted()) {
+            $errors = $formHandler->save($emailConfigurationForm->getData());
+
+            if (empty($errors)) {
```

Comment:
Do the same as `processDeleteSelectedEmailLogsAction` keep the same order please :)